src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -91,6 +114,10 @@ private void clearCommand() {
         updateCurrentCommandField();
     }
 
+    private void runCommand(){
+        commandManager.getCurrentCommand().execute(driverManager.getCurrentDriver());
+    }
```

Comment:
go for independency from DriverManager. Move this method to CommandManager and extract CommandManager interface (DIP principle)